### DISCUSSION TITLE
DOES ASSET UPDATE WORK, AND IF SO HOW

### DISCUSSION THREAD

**Discussion by barte525** [barte525/RSI#52] [Announcements]
"On asset put(I want to update asset), when I add ID to body I got false in response and nothing happens, when I don't add ID to body new asset is creating. I think it worked but I don't know how or it stopped working, can someone test it?"

**Comment by ale-gen** [comment]
While testing I could update asset with body contains all attributes. In this case everything was nice. But updating asset without id in body, there was created new asset.
